spec/mailers/course_create_email_mailer_spec.rb

```diff
@@ -31,6 +31,14 @@
       expect(mail.govuk_notify_personalisation[:course_code]).to eq(course.course_code)
     end
 
+    it "includes the course description in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:course_description]).to eq(course.description)
+    end
+
+    it "includes the course funding type in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:course_funding_type]).to eq(course.funding_type)
+    end
+
     it "includes the datetime for the creation in the personalisation" do
       expect(mail.govuk_notify_personalisation[:create_course_datetime]).to eq("4:05am on 3 February 2001")
     end
```

Comment:
Could you add a test for the afternoon time format? just so we don't end up questioning this again

Reply:
Yep - now added to `time_format_spec.rb`